components/Header/NavDeskop.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
przyjmijmy 1 strukturę folderów, bo tu się robi chaos.
każdy komponent znajduje się w swoim folderze, czyli
1) `components/header/Header.tsx`
- foldery piszemy z małej litery, komponenty z dużej
2) w tym wypadku jeśli NavDesktop (masz literówke) jest w strukturze headera, to jego ścieżka będzie taka:
`components/header/navDesktop/NavDesktop.tsx'

3) popraw proszę wszystkie foldery i pliki żeby były zgodnie z taką konwencją

Reply:
zrobione